bukkit/src/main/java/net/william278/husktowns/gui/GuiSettings.java

```diff
@@ -0,0 +1,170 @@
+/*
+ * This file is part of HuskTowns, licensed under the Apache License 2.0.
+ *
+ *  Copyright (c) William278 <will27528@gmail.com>
+ *  Copyright (c) contributors
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package net.william278.husktowns.gui;
+
+import com.google.gson.annotations.Expose;
+import org.bukkit.ChatColor;
+import org.bukkit.Material;
+import xyz.xenondevs.invui.item.ItemProvider;
+import xyz.xenondevs.invui.item.builder.ItemBuilder;
+
+import java.util.Arrays;
+import java.util.Map;
+
+import static java.util.Map.entry;
+
+public class GuiSettings {
+    private static GuiSettings instance = new GuiSettings();
```

Comment:
I'd like to broadly avoid any static variable use - let's store this in the main GuiManager